presto-main/src/test/java/io/prestosql/operator/scalar/TestMathFunctions.java

```diff
@@ -1190,8 +1190,8 @@ public void testGreatest()
         assertFunction("greatest(1.0E0, 2)", DOUBLE, 2.0);
         assertFunction("greatest(1, 2.0E0)", DOUBLE, 2.0);
         assertFunction("greatest(1.0E0, 2)", DOUBLE, 2.0);
-        assertFunction("greatest(5.0E0, 4, CAST(NULL as DOUBLE), 3)", DOUBLE, 5.0);
-        assertFunction("greatest(5.0E0, 4, CAST(NULL as BIGINT), 3)", DOUBLE, 5.0);
+        assertFunction("greatest(5.0E0, 4, CAST(NULL as DOUBLE), 3)", DOUBLE, null);
+        assertFunction("greatest(5.0E0, 4, CAST(NULL as BIGINT), 3)", DOUBLE, null);
```

Comment:
Duplicate test case?

Reply:
Duplicate with what? They are different input types

Reply:
The two lines
```
assertFunction("greatest(5.0E0, 4, CAST(NULL as DOUBLE), 3)", DOUBLE, null);
assertFunction("greatest(5.0E0, 4, CAST(NULL as BIGINT), 3)", DOUBLE, null);
```

are probably equivalent from `greatest` function perspective (with and without a coercion, but does that matter?)